netlify.toml

```diff
@@ -1,6 +1,14 @@
 [build]
-    publish = "public"
-
+  publish = "public"
```

Comment:
netlify.toml supports different sections for the different contexts already.

https://github.com/gohugoio/hugoDocs/blob/master/netlify.toml#L11

Hugo best practices say that `baseURL` should be set to "https://istio.io/" (probably with `/latest/`?  I don't quite recall where that is added)

From a quick read of the docs, the environment in development _should_ use localhost instead of the `baseURL`.